tool/tsh/db.go

```diff
@@ -728,6 +729,9 @@ func onDatabaseConnect(cf *CLIConf) error {
 	if routeToDatabase.Protocol == defaults.ProtocolDynamoDB {
 		return trace.BadParameter(formatDbCmdUnsupportedDBProtocol(cf, routeToDatabase))
 	}
+	// if routeToDatabase.Protocol == defaults.ProtocolElasticsearch && routeToDatabase.Username == "" {
```

Comment:
If this is not necessary then lets remove it. No need to keep commented-out code.

Reply:
Oops, my bad. This wasn't meant to be here.